workspace/apps/pidp/src/scss/material/_material-overrides.scss

```diff
@@ -43,6 +43,13 @@
   }
 }
 
+.viewport-xsmall {
```

Comment:
Seems like the tablet view is still breaking. Can you add the `viewport-small` selector to this as well?